.github/dependabot.yml

```diff
@@ -3,8 +3,13 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: daily
+    interval: monthly
+  ignore:
+    - dependency-name: "@types/node"
+    - dependency-name: "*"
+      update-types: ["version-update:semver-patch"]
+  open-pull-requests-limit: 3
```

Comment:
I would suggest raising the limit. Otherwise, whenever a pr is merged, dependabot will open a new pr (when there are multiple packages that need to be updated)
```suggestion
  open-pull-requests-limit: 20
```